.gitlab-ci.yml

```diff
@@ -195,3 +196,28 @@ examples-generate-metadata:
         cargo contract generate-metadata &&
         popd;
       done
+
+#### stage:                        publish
+
+publish-docs:
+  stage:                           publish
+  <<:                              *docker-env
+  only:
+    - master
+    - schedules
+    - tags
```

Comment:
Not sure we need this outside merged to `master` tbh

Reply:
- `schedules` can be a nightly job on `master`, we just didn't need it before.
- `tags` will do a documentation update when you release.
Don't you need any of those?